### ISSUE TITLE
multiword headword tagging issues

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/ACC#16]
Look at the following entry in acc6.txt
```
<L>29478<pc>1-756,1<k1>haridIkzita<k2>hari dIkzita
{#hari dIkzita#}¦ <ab type="pers">son</ab> of <ab type="hw" value="vIreSvara">Vīreśvara</ab> Dīkṣita, <ab type="pers">grandson</ab> of
<>Bhaṭṭoji Dīkṣita, <ab type="pers">pupil</ab> of <ab type="hw" value="rAmASrama">Rāmāśrama</ab> (<ls>IO</ls>. 1346),
```
The corresponding entries are
```
<L>23172<pc>1-596,1<k1>vIreSvaradIkzita<k2>vIreSvara dIkzita
{#vIreSvara dIkzita#}¦ <ab type="pers">son</ab> of Bhaṭṭoji Dīkṣita, <ab type="pers">father</ab> of <ab type="hw" value="hari">Hari</ab>
<>Dīkṣita (<ab type="hw" value="Sabdaratna">Śabdaratna</ab>).
<LEND>
<L>23173<pc>1-596,1<k1>vIreSvara<k2>vIreSvara
{#vIreSvara#}¦ <ab type="pers">son</ab> of <ab type="hw" value="lakzmaRa">Lakṣmaṇa</ab>, <ab type="pers">father</ab> of <ab type="hw" value="veRIdatta">Veṇīdatta</ab> (<ab type="hw" value="alaMkAracandrodaya">Alaṃ-
<>kāracandrodaya</ab>). <ls>IO</ls>. 235.
<LEND>
```

Currently because we split at every word boundary, vIreSvara dIkzita was not tagged properly. It was tagged to vIreSvara who is somebody else.
Similar situation may arise for Bawwa, paRqita, some other honorific / common surname.

**Comment by gasyoun** [comment]
> Currently because we split at every word boundary

How can we otherwise?

**Comment by drdhaval2785** [comment]
Good old bigrams, trigrams etc. Because abbrvs ot headwords of interest span more than one word, our approach should also span more.

**Comment by gasyoun** [comment]
> Good old bigrams, trigrams 

I thought bigrams are dead. I felt I was wrong. Now I know why.

**Comment by funderburkjim** [comment]
We seem to run into this problem in many dictionaries whose digitization follows the line breaks of the
printed text. 
On the one hand, preserving these line breaks in the digitization is quite helpful for the purpose of comparing the digitization to the text.

On the other hand, preserving these line breaks in the digitization is a significant obstacle when we use markup to expose the semantic structure of the entries.

**Comment by gasyoun** [comment]
> On the one hand, preserving these line breaks in the digitization is quite helpful for the purpose of comparing the digitization to the text.
On the other hand, preserving these line breaks in the digitization is a significant obstacle when we use markup to expose the semantic structure of the entries.

Killing them would not be wise. But if we can replace with a markup, that remembers the line breaks - that will be quite enough.

**Comment by funderburkjim** [comment]
> replace with a markup, that remembers the line breaks

A good principle.  However, as with other internal markup, doing this in a systematic way, similar across dictionaries, is currently slightly beyond my/our understanding.  Thus, I'm not really attempting this now.
Dhaval's approach in part of his ACC work seemed fine for the particular situation he encountered in that particular dictionary.  The issues are different in other dictionaries.  Right now, it's just of interest to notice the kinds of problems that line breaks introduce.  At some time, we will understand these problems sufficiently to code around them.

**Comment by gasyoun** [comment]
> At some time, we will understand these problems sufficiently to code around them.

Agree, now it would be enough to have one place to document all such peculiarities, so we do not forget ourself about them in a year or two.
